source/features/clean-conversation-sidebar.tsx

```diff
@@ -93,9 +93,12 @@ async function clean(): Promise<void> {
 	}
 
 	// Labels
-	if (!cleanSection('.sidebar-labels') && !canEditSidebar()) {
-		// Hide header in any case except `canEditSidebar`
-		select('.sidebar-labels div.discussion-sidebar-heading')!.remove();
+	if (!cleanSection('.js-issue-labels') && !canEditSidebar()) {
+		// Hide heading in any case except `canEditSidebar`
+		select('.js-issue-labels')!
+			.closest('.discussion-sidebar-item')!
+			.querySelector(':scope > .discussion-sidebar-heading')!
+			.remove();
```

Comment:
```suggestion
		select('[data-hotkey="l"]')!.remove();
```

Reply:
Tested this change: the hotkeys are only there if the user has admin privileges in the repo, so we can't fully rely on that selector.

Reply:
It might be worth leaving a comment, I think we talked about this before

Reply:
Should we still use this selector and keep the "chain" as a fallback? Or just use the chain regardless?

Reply:
Shorter selectors make sense if they shorten the code. We don't duplicate code for negligible speed improvements.